orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java

```diff
@@ -73,4 +76,32 @@ public InstanceStatusResponse getInstance(@PathParam("instanceId") String instan
         Map<HostName, String> hostStatusStringMap = OrchestratorUtil.mapValues(hostStatusMap, HostStatus::name);
         return InstanceStatusResponse.create(applicationInstance, hostStatusStringMap);
     }
+
+    @GET
+    @Path("/{instanceId}/slobrok")
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<SlobrokEntryResponse> getSlobrokEntries(
+            @PathParam("instanceId") String instanceId,
+            @QueryParam("pattern") String pattern) {
+        ApplicationInstanceReference reference = parseInstanceId(instanceId);
+        ApplicationId applicationId = OrchestratorUtil.toApplicationId(reference);
+
+        if (pattern == null) {
+            pattern = "**";
+        }
+
+        List<Mirror.Entry> entries = slobrokMonitorManager.lookup(applicationId, pattern);
+        return entries.stream().map(SlobrokEntryResponse::fromMirrorEntry)
+                .collect(Collectors.toList());
+    }
+
+    private ApplicationInstanceReference parseInstanceId(@PathParam("instanceId") String instanceIdString) {
+        ApplicationInstanceReference instanceId;
+        try {
+            instanceId = parseAppInstanceReference(instanceIdString);
```

Comment:
Consider removing `instanceId` variable and return result from `parseAppInstanceReference` directly.

Reply:
Done